app/js/components/setsOrderEntry/OrderSetForm.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import SelectBox from '../selectBox';
+import Accordion from '../orderEntry/Accordion';
+import OrderHeader from '../orderEntry/OrderHeader';
+import OrderSetDetails from './OrderSetDetails';
+import './styles.scss';
+
+
+class OrderSetForm extends PureComponent {
+  state = {
+    items: [{ value: 'Choose a set to add', id: 0 }, { value: 'Mineral', id: 1 }],
+    selectedItem: '',
+    isDisabled: false,
+    displaySets: 'block',
+    displayForm: 'none',
+  }
+
+  onSelectOrderSet = () => {
```

Comment:
Please rename this as a function, `onSelectOrderSet` sounds like an event handler.

Reply:
Done